File: okj/rendszerüzemeltetö/hianyzasok/java/Hianyzas.java
public class Hianyzas{
    
    public final String nev;
    public final String osztaly;
    public final int elsoNap;
    public final int utolsoNap;
    public final int mulasztottOrak;
    
    public Hianyzas(String sor) {
        var split = sor.split(";");
        
        this.nev = split[0];
        this.osztaly = split[1];
        this.elsoNap = Integer.parseInt(split[2]);
        this.utolsoNap = Integer.parseInt(split[3]);
        this.mulasztottOrak = Integer.parseInt(split[4]);
    }
}

File: okj/rendszeruzemelteto/hianyzasok/c#/Hianyzas.cs
public class Hianyzas {

    public readonly string nev;
    public readonly string osztaly;
    public readonly int elsoNap;
    public readonly int utolsoNap;
    public readonly int mulasztottOrak;

    public Hianyzas(string sor) {
        var split = sor.Split(';');

        this.nev = split[0];
        this.osztaly = split[1];
        this.elsoNap = int.Parse(split[2]);
        this.utolsoNap = int.Parse(split[3]);
        this.mulasztottOrak = int.Parse(split[4]);
    }
}